src/Controllers/SedimentController.ts

```diff
@@ -90,6 +90,8 @@ module StreamStats.Controllers {
         private init(): void {
             this.isBusy = false;
 
+            this.selectedReferenceGage = new Models.ReferenceGage("", ""); 
```

Comment:
First check to see if this.studyAreaService.selectedStudyArea.sedimentModelData.selectedGage is null or not